x/auth/types/codec.go

```diff
@@ -25,8 +24,6 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&ModuleCredential{}, "cosmos-sdk/GroupAccountCredential", nil)
 
 	legacy.RegisterAminoMsg(cdc, &MsgUpdateParams{}, "cosmos-sdk/x/auth/MsgUpdateParams")
-
-	legacytx.RegisterLegacyAminoCodec(cdc)
```

Comment:
I think you removed the call to this function, but you can remove the function too